.env.j2

```diff
@@ -0,0 +1,13 @@
+FLASK_APP = todo_app/app
+FLASK_ENV = production
+
+SECRET_KEY = {{ secret_key }}
+
+API_BASE_URL = https://api.trello.com
+
+BOARD_ID = 634939e098d70700c50f8477

```

Comment:
Even though these are not very sensitive (i.e. they're useless by themselves without the API_KEY and TOKEN), I would still treat them as secrets and hide them.